week-1/2-mandatory/3-function-output.js

```diff
@@ -1,16 +1,22 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// This function generate a random number between a range of 0 to 10.
+// Math.random() generate a floating point number between a range of 0 to 1. 
+// In which 0 is inclusive and 1 is exclusive.
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// concat() method is used to combine or join two strings.
+// In this function w2 is added to the infront of w1.
```

Comment:
This is almost right! If you try running this function by providing string inputs `w1` and `w2` where does w2 go?